<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QSpinBox" name="sbShoulders">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>111</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>shoulder </string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbLegs">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>111</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>leg </string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbFoots">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>111</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>foot </string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbHead">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>111</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>head </string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
         <widget class="QComboBox" name="cbAttitude">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>86</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>relax</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>standing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>walking</string>
           </property>
          </item>
         </widget>
         <widget class="QDoubleSpinBox" name="dspHeight">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>height </string>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="minimum">
           <double>60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>150.000000000000000</double>
          </property>
          <property name="value">
           <double>120.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspGroundRoll">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>gnd roll </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspGroundPitch">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>50</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>gnd pitch </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspBodyRoll">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>body roll </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspBodyPitch">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>body pitch </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspBodyYaw">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>260</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>body yaw </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbConnect">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>370</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>connect</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="dspGroundYaw">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>80</y>
            <width>161</width>
            <height>26</height>
           </rect>
          </property>
          <property name="prefix">
           <string>gnd yaw </string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="IPM_Plot3D_Widget" name="widget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IPM_Plot3D_Widget</class>
   <extends>QWidget</extends>
   <header location="global">IPM_Plot/IPM_Plot3D_Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
